R/plotting/access_change_area.R

```diff
@@ -0,0 +1,61 @@
+library("tidyverse")
+library("config")
+library("here")
```

Comment:
Remove quotation marks from packages: `library(tidyverse)`
This comment goes for all plotting functions.

Reply:
I've actually never figured out which way is better.. and probably therefore I've used both in same PR. :smiley: